zerver/views/user_settings.py

```diff
@@ -191,6 +192,10 @@ def update_display_settings_backend(request, user_profile,
         do_change_emoji_alt_code(user_profile, emoji_alt_code)
         result['emoji_alt_code'] = emoji_alt_code
 
+    elif (emojiset is not None and user_profile.emojiset != emojiset):
+        do_change_emojiset(user_profile, emojiset)
+        result['emojiset'] = emojiset
+
```

Comment:
If you rebase, you'll see that one can now use `do_change_user_display_setting` (or some name like that) for this and avoid needing to add code in `actions.py` at all.